src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const basePrice = days * PRICE;
+
+  switch (true) {
+    case days >= LONG_TERM:
+      return basePrice - LONG_TERM_DISCOUNT;
+    case days >= SHORT_TERM:
+      return basePrice - SHORT_TERM_DISCOUNT;
+    default:
+      return basePrice;
```

Comment:
Using a switch statement with `true` as the expression and conditions as case clauses is unconventional and can be confusing. It's better to use if-else statements here, as they are more straightforward for this kind of conditional logic.

Reply:
better to use an if statement here
<img width="722" alt="image" src="https://github.com/mate-academy/js_task-transportation-on-vacation/assets/70220667/246cc6cc-0694-4b78-893e-fd5b01301777">